client/state/account-recovery/test/reducer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ACCOUNT_RECOVERY_SETTINGS_FETCH,
+	ACCOUNT_RECOVERY_SETTINGS_FETCH_SUCCESS,
+	ACCOUNT_RECOVERY_SETTINGS_FETCH_FAILED,
+} from 'state/action-types';
+
+import reducer from '../reducer';
+
+describe( '#account-recovery/isFetchingSettings reducer :', () => {
+	it( 'should set isFetchingSettings flag.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_SETTINGS_FETCH,
+		} );
+
+		assert.isTrue( state.isFetchingSettings );
+	} );
+
+	it( 'should unset isFetchingSettings flag on success.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_SETTINGS_FETCH_SUCCESS,
+			settings: {
+				email: '',
+				email_validated: true,
+				phone: {},
+				phone_validated: true,
```

Comment:
The meaning behind these two flags is that we will require validating of the ownership and accessibility of recovery emails and phones. In case of recovery phones, once a user adds / updates ones recovery phone, it is considered "unvalidated" and `phone_validated` will be `false`. After going through the validation process, it is "validated" and `phone_validated` will be `true`.

If I understand your meaning correctly, I think it is more of the former, is it?